calculate_error.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
```

Comment:
Dodałabym import:
`from sklearn.metrics import mean_absolute_error, mean_squared_error`

i potem już tego używała w głównej funkcji.
I jeżeli tak to usunęłabym poniższe implementacje.